src/components/forms/TextField.js

```diff
@@ -60,11 +55,17 @@ export default {
       const inputLength = (this.inputValue && this.inputValue.toString() || '').length
       let min = inputLength
 
-      if (this.min !== 0 && inputLength < this.min) {
-        min = this.min
+      if (this.counterMin !== 0 && inputLength < this.counterMin) {
```

Comment:
You compare this to an absolute equal of 0, but since a string can be passed, it could be '0'.